### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Finally got it to work lol

File: stdlib/FoundationKit/String.c [hunk 1]
```diff
@@ -1,26 +1,71 @@
...
+
 ClassInitFunction(String)
     ClassMethod(String, ":create", foundation_string_new)
     ClassMethod(String, ":value", foundation_string_value)
-EndClassInitFunction
\ No newline at end of file
+    ClassMethod(String, ":length", foundation_string_length)
+    ClassMethod(String, ":append:with", foundation_string_append)
+    ClassMethod(String, ":copy", foundation_string_copy)
+    ClassMethod(String, ":at", foundation_string_at)
```
Comment (sowelipililimute) [inline stdlib/FoundationKit/String.c]:
```suggestion
    ClassMethod(String, ":character:at", foundation_string_at)
```

```
(`string` character at: 5)
```

File: stdlib/FoundationKit/String.c [hunk 2]
```diff
@@ -1,26 +1,71 @@
...
+    ClassMethod(String, ":set:at", foundation_string_set_at)
```
Comment (sowelipililimute) [inline stdlib/FoundationKit/String.c]:
```suggestion
    ClassMethod(String, ":set:character:at:to", foundation_string_set_at)
```

```
(`string` set character at: 5 to: `a`)
```

File: stdlib/FoundationKit/String.c [hunk 3]
```diff
@@ -1,26 +1,71 @@
...
+{
+    return foundation_string_new(foundation_string_length(self));
+}
+
+char foundation_string_at(Object* self, size_t index)
+{
+    return ((StringObject*)self)->data[index];
+}
+
+void foundation_string_set_at(Object* self, char character, size_t index)
```
Comment (sowelipililimute) [inline stdlib/FoundationKit/String.c:57]:
```suggestion
void foundation_string_set_at(Object* self, size_t index, char character)
```

**Review by sowelipililimute — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Any other issues?

**Pull request merged**